src/swap/components/Swap.tsx

```diff
@@ -50,10 +53,14 @@ export function Swap({
         )}
         data-testid="ockSwap_Container"
       >
-        <div className="mb-4">
-          <h3 className={text.title3} data-testid="ockSwap_Title">
+        <div className="mb-4 flex items-center justify-between">
+          <h3
+            className={cn(text.title3, 'text-inherit')}
+            data-testid="ockSwap_Title"
+          >
             {title}
           </h3>
+          <div>{swapSettings}</div>
```

Comment:
So we need this `<div>`? Can we avoid to have it?

Reply:
good call, removed